src/main/java/core/basesyntax/classes/figures/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.classes.figures;
+
+public class RightTriangle extends Figure {
+    private final int firstLeg;
+    private final int secondLeg;
+
+    public RightTriangle(String color, int firstLeg, int secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.